src/public.js

```diff
@@ -10,5 +10,7 @@ function publishExternalAPI() {
   }]).directive({
     draggable: draggableDirective,
     droppable: droppableDirective
+  }).factory({
+    $dndPosition: $positionFactory
```

Comment:
I might suggest renaming this to $dndDOM or something, because it will probably do more than just deal with position